server/command/command.go

```diff
@@ -7,10 +7,9 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/pkg/errors"
-
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/plugin"
+	"github.com/pkg/errors"
```

Comment:
This seems weird. We usually have the non-Mattermost in groups above the Mattermost packages. WIll the linter allow you to order/group dependencies like this?

```go
import (
    "strings"

    "github.com/pkg/errors"	

    "github.com/mattermost/mattermost-server/v5/model"
    "github.com/mattermost/mattermost-server/v5/plugin"

    "github.com/mattermost/mattermost-plugin-mscalendar/server/config"
)
```

Reply:
This should be applied to all Go files that are applicable

Reply:
> We usually have the non-Mattermost in groups above the Mattermost packages. 

is that really the convention? I would argue that treat `github.com/mattermost/mattermost-server/v5` in the same way as other library. Also, we don't enforce any rules around this making it hard to check.